proto/desmos/tokenfactory/v1beta1/msgs.proto

```diff
@@ -0,0 +1,141 @@
+syntax = "proto3";
+package desmos.tokenfactory.v1beta1;
+
+import "gogoproto/gogo.proto";
+import "cosmos/base/v1beta1/coin.proto";
+import "cosmos/bank/v1beta1/bank.proto";
+import "cosmos_proto/cosmos.proto";
+import "cosmos/msg/v1/msg.proto";
+
+import "desmos/tokenfactory/v1beta1/models.proto";
+
+option go_package = "github.com/desmos-labs/desmos/v4/x/tokenfactory/types";
+
+// Msg defines the tokefactory module's gRPC message service.
+service Msg {
+  option (cosmos.msg.v1.service) = true;
+
+  // CreateDenom allows an account to create a new denom for subspace. It
+  // requires a subspace and a sub denomination. The (subspace_treasury_address,
+  // sub_denomination) tuple must be unique and cannot be re-used.
+  //
+  // The resulting denom created is defined as
+  // <factory/{treasuryAddress}/{subdenom}>. The resulting denom's admin is
+  // originally set to be the subspace treasury account, and this can not be
+  // changed later.
```

Comment:
To pass the proto-lint check, I moved the document from request type to here. The original one is here:
https://github.com/CosmWasm/token-factory/blob/main/proto/osmosis/tokenfactory/v1beta1/tx.proto#L24